packages/models/dist/index.d.ts

```diff
@@ -0,0 +1,32 @@
+declare class User {
+    id: string;
```

Comment:
we do not need dist folder, see how ui package is done

Reply:
ok

Reply:
Removed